### DISCUSSION TITLE
Create client with access-type=bearer-only via ui

### DISCUSSION THREAD

**Discussion by alexgordtop** [keycloak/keycloak#23376] [Q&A]
Hello together,

in older keycloak versions it was possible to create a client with access-type=bearer-only  - instead of public or confidential.
I explicitly want to receive a 401 when the backend tries to verify a token that is no longer valid.
I don't want to be redirected somewhere - since the rest service is not capable of that.

The documentation lists that option, but I can't find it in ui: https://www.keycloak.org/docs/22.0.3/server_admin/#_audience_resolve
https://www.keycloak.org/docs/22.0.3/server_admin/#architecture

There seem to be a keycloak java adapter option "bearer-only":
https://www.keycloak.org/docs/22.0.3/securing_apps/#_java_adapter_config

But from my understanding, this is also a client setting...

Thanks in advance,
Alex

**Comment by justin-tay** [comment]
That terminology has been removed. I think a bearer-only client is now a client with no authentication flows enabled.

- https://github.com/keycloak/keycloak/pull/23148

**Comment by alexgordtop** [comment]
@justin-tay  Thanks for the hint.
